openwisp_controller/config/static/sortedm2m/patch_sortedm2m.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+(function ($) {
+    $(document).ready(function () {
+        if ($('.sortedm2m-items').length < 2) {
+            destroyHiddenSortableWidget($);
+        }
+        $(document).on('click', 'a.inline-deletelink', function () {
+            destroyHiddenSortableWidget($);
+        });
```

Comment:
I found that the previous solution was not working when the config form was removed and then added back using the "add row" button. 

This solution is also not ideal because it is listening for all `.inline-deletelink` elements. I had to choose this because [inline.js in Django has `event.preventDefault` present in the click handler](https://github.com/django/django/blob/222bf2932b55ebc964ffc5f9a6f47bad083e5ac2/django/contrib/admin/static/admin/js/inlines.js#L121-L122).